packages/tools/devtools/devtools-view/src/test/SettingsView.test.tsx

```diff
@@ -0,0 +1,20 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+// eslint-disable-next-line import/no-unassigned-import
```

Comment:
Nit, so it's clear why the disable
```suggestion
// eslint-disable-next-line import/no-unassigned-import - Normal usage pattern for @testing-library/jest-dom
```